### ISSUE TITLE
Error when debug a simple code

### ISSUE THREAD

**Issue by brxxlstxrs** [microsoft/debugpy#1013]
## Environment data

- debugpy version: 1.6.2
- OS and version: Arch Linux x84_64
- Python version: 3.10.5
- Using Neovim with dap:

## Expected behavior

```
hello
```

## Actual behavior

```
0.51s - Error inserting pydevd breaks.
Traceback (most recent call last):
  File "/home/misha/.virtualenvs/debugpy/lib/python3.10/site-packages/debugpy/_vendored/pydevd/_py
devd_frame_eval/pydevd_modify_bytecode.py", line 328, in insert_pydevd_breaks
    for new_instruction in get_instructions_to_add(
  File "/home/misha/.virtualenvs/debugpy/lib/python3.10/site-packages/debugpy/_vendored/pydevd/_py
devd_frame_eval/pydevd_modify_bytecode.py", line 102, in get_instructions_to_add
    Instr("LOAD_CONST", _pydev_stop_at_break, lineno=stop_at_line - 1),
  File "/home/misha/.virtualenvs/debugpy/lib/python3.10/site-packages/debugpy/_vendored/pydevd/_py
devd_frame_eval/vendored/bytecode/instr.py", line 171, in __init__
    self._set(name, arg, lineno)
  File "/home/misha/.virtualenvs/debugpy/lib/python3.10/site-packages/debugpy/_vendored/pydevd/_py
devd_frame_eval/vendored/bytecode/instr.py", line 239, in _set
    _check_lineno(lineno)
  File "/home/misha/.virtualenvs/debugpy/lib/python3.10/site-packages/debugpy/_vendored/pydevd/_py
devd_frame_eval/vendored/bytecode/instr.py", line 74, in _check_lineno
    raise ValueError("invalid lineno")
ValueError: invalid lineno
```

## Steps to reproduce:
Have installed debugpy with:
```
mkdir ~/.virtualenvs
cd ~/.virtualenvs
python -m venv debugpy
debugpy/bin/python -m pip install debugpy
```

Dap config:
```lua
require('dap-python').setup('~/.virtualenvs/debugpy/bin/python')
table.insert(require('dap').configurations.python, {
  justMyCode = false,
})

require("nvim-dap-virtual-text").setup()

-- Keymaps
vim.keymap.set("n", "<leader>b", ":lua require'dap'.toggle_breakpoint()<cr>")
vim.keymap.set("n", "<leader>c", ":lua require'dap'.continue()<cr>")
vim.keymap.set("n", "<leader>o", ":lua require'dap'.step_over()<cr>")
vim.keymap.set("n", "<leader>i", ":lua require'dap'.step_into()<cr>")
vim.keymap.set("n", "<leader>d", ':lua require("dapui").toggle()<cr>')
```

Do:
1. Set breakpoint on first line with setting cursor on firsy line and do `:lua require("dapui").toggle_breakpoint()`
2. Open dap-ui with `:lua require("dapui").toggle()`
3. Do `:lua require'dap'.continue()`
4. See error in dapui console

(Already open issue in official dap repo: https://github.com/mfussenegger/nvim-dap/issues/645)

**Comment by fabioz** [comment]
This is actually already fixed in `main` (in https://github.com/microsoft/debugpy/commit/a294092d9c6d8459126ecb8f537b6012fb7e7d28).

As a note, this would only happen in frame eval mode, so, it's possible to disable it through an environment variable: `PYDEVD_USE_FRAME_EVAL=0`

**Comment by brxxlstxrs** [comment]
> This is actually already fixed in `main` (in [a294092](https://github.com/microsoft/debugpy/commit/a294092d9c6d8459126ecb8f537b6012fb7e7d28)).
> 
> As a note, this would only happen in frame eval mode, so, it's possible to disable it through an environment variable: `PYDEVD_USE_FRAME_EVAL=0`

Plz tell how set `PYDEVD_USE_FRAME_EVAL=0` variable?
